extractors/README.md

```
@@ -0,0 +1,89 @@
+
+### Core Types
+
+- `Extractor`: Core extraction function with metadata
+- `Source`: Enumeration of extraction sources (Header, AuthHeader, Query, Form, Param, Cookie, Custom)
+- `ErrNotFound`: Standardized error for missing values
+
+### Extractor Structure
+
+```go
+type Extractor struct {
+	Extract    func(fiber.Ctx) (string, error)
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)